apps/meteor/server/startup/migrations/v299.ts

```diff
@@ -0,0 +1,27 @@
+import { Settings } from '@rocket.chat/models';
+
+import { settingsRegistry } from '../../../app/settings/server';
+import { addMigration } from '../../lib/migrations';
+
+addMigration({
+	version: 299,
+	async up() {
+		const customOauthServices = await Settings.find({ _id: /Accounts_OAuth_Custom-[^-]+$/im }).toArray();
+		const serviceNames = customOauthServices.map(({ _id }) => _id.replace('Accounts_OAuth_Custom-', ''));
+
+		serviceNames.forEach(async (serviceName) => {
+			await settingsRegistry.add(`Accounts_OAuth_Custom-${serviceName}-merge_users_distinct_services`, '', {
```

Comment:
the default value is `false` right? I didn't check every other field
```suggestion
			await settingsRegistry.add(`Accounts_OAuth_Custom-${serviceName}-merge_users_distinct_services`, false, {
```

Reply:
Oops, fixed! I rechecked the other ones and they're ok. Tkss